lib/rules/web/admin_console/4.6/clustersettings.xyaml

```diff
@@ -30,3 +30,17 @@ click_update_channel_button:
   - selector:
       xpath: //button[@data-test-id='current-channel-update-link']
     op: click
+click_channels_drop_down:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-button']
+    op: click
+check_drop_down_items: {}
```

Comment:
Why `check_drop_down_items` is empty?

Reply:
@yapei update:
- corrected it. I was thinking for released versions this would be empty but now I see it will not.